library/cc/engine_builder.cc

```diff
@@ -190,8 +214,24 @@ EngineBuilder& EngineBuilder::enableH2ExtendKeepaliveTimeout(bool h2_extend_keep
   return *this;
 }
 
+EngineBuilder& EngineBuilder::setPorts(std::vector<uint32_t> ports) {
+  this->ports_ = ports;
+  return *this;
+}
+
+EngineBuilder& EngineBuilder::setIpvVersion(std::string ipv_version) {
+  this->ipv_version_ = ipv_version;
+  return *this;
+}
+
+EngineBuilder& EngineBuilder::enableCustomTransportSocket(bool enable_transport_socket) {
+  this->enable_transport_socket_ = enable_transport_socket;
+  return *this;
+}
+
 EngineBuilder&
 EngineBuilder::enablePlatformCertificatesValidation(bool platform_certificates_validation_on) {
+
```

Comment:
please do go through and revert things like this and newlines that aren't necessary changes - just makes review easier.